modules/cf-core/src/methods/app-instance/propose-install.ts

```diff
@@ -77,6 +78,7 @@ export class ProposeInstallAppInstanceController extends NodeController {
 
     await protocolRunner.initiateProtocol(ProtocolNames.propose, {
       ...params,
+      stateTimeout: stateTimeout || Zero,
```

Comment:
Should be stateTimeout or `undefined` (and if `undefined`, it defaults to `defaultTimeout`